Apps/phironnet/ironnet.json

```diff
@@ -41,6 +41,48 @@
             "data_type": "password",
             "required": true,
             "order": 3
+        },
+        "enable_alert_notifications": {
+            "description": "Enable Alert Notifications",
+            "data_type": "boolean",
+            "required": false,
+            "default": true,
+            "order": 10
+        },
```

Comment:
Add a configuration for the alert_action(s) to consume - defaulting to just ENA_ALERT_CREATED.